src/main/java/net/dries007/tfc/client/ClientForgeEventHandler.java

```diff
@@ -189,6 +190,12 @@ public static void onItemTooltip(ItemTooltipEvent event)
             stack.getCapability(HeatCapability.CAPABILITY).ifPresent(cap -> cap.addTooltipInfo(stack, text));
             stack.getCapability(EggCapability.CAPABILITY).ifPresent(cap -> cap.addTooltipInfo(text));
 
+            ItemStack bait = TFCFishingRodItem.getBaitItem(stack);
+            if (!bait.isEmpty())
+            {
+                text.add(new TranslatableComponent("tfc.tooltip.fishing.bait").append(bait.getHoverName()));
+            }
```

Comment:
This should be in `addInformation()` since it's specific to `TFCFishingRodItem`s